### ISSUE TITLE
Spike: Investigate integrating Pension submission process with BPDS system

### ISSUE THREAD

**Issue by sanjabaj2** [department-of-veterans-affairs/va.gov-team#92763]
## Issue Description
Take a look at attached slides describing current process for pension application, requiring OCR to extract data. During investigation of overflow problem, we identified an existing system called BPDS that is designed to accept raw json content for individual forms. 
BPDS stands for Benefits Process Data Service.

The spec can be [found here](https://bpds-dev.dev.bip.va.gov/swagger-ui.html#/).  BPDS is a repository designed to house JSON versions of scanned VA forms.
The plan would be to send json to BPDS and send PDF directly to eFolder. 
While ticket is for Pension, the proposed plan / design should work for any other forms. 

These are the links to various environments that they have
https://bpds-dev.dev.bip.va.gov/swagger-ui.html#/
https://bpds-test.dev.bip.va.gov/swagger-ui.html#/
https://bpds-uat.stage.bip.va.gov/swagger-ui.html#/

Before this work starts, we may need to get an approval from VA security teams. Sanja is looking into it. 

Link to list of BPDS json schemas for various forms:
https://github.com/department-of-veterans-affairs/bip-governance/tree/3b32694ca8f4deec70cb6c7f49b4ca53af7a5360/Data-Structures

Link to VBMS Documents API for uploading PDF to eFolder:
https://developer.va.gov/explore/api/benefits-documents

[BPDS Integration.pdf](https://github.com/user-attachments/files/16984476/BPDS.Integration.pdf)
## Tasks
- [ ] Document steps needed to integrate with BPDS
- [ ] Document steps / options to load test BPDS system. How many forms per second can they take? 
- [ ] Create tickets to the actual integration. 
- [ ] Build integration prototype (maybe this is a separate ticket?)

## Acceptance Criteria
- [ ] We have a blue print of the work needed to complete this integration. 
---
## How to configure this issue
- [ ] **Attached to an Epic** (This work will become part of an Epic, once it is ready for implementation)

**Comment by matt4su** [comment]
@sanjabaj2 tagging you following up on  our discussion to include your thoughts on the end state this work will ultimately facilitate.

**Comment by sanjabaj2** [comment]
Ideal - final state is that we send json to BPDS and just upload PDF to eFolder via BEP Claims Evidence API https://claimevidence-api-test.dev.bip.va.gov/api/v1/rest/swagger-ui.html

But as interim step we may need to continue to send PDF via Benefits intake until MAS and other system catch up.
